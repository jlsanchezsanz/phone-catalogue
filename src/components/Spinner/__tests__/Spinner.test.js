import React from 'react';
import { shallow } from 'enzyme';

import Spinner from '../Spinner';

test('should render component', () => {
  const component = shallow(<Spinner />);
  expect(component).toMatchSnapshot();
});
