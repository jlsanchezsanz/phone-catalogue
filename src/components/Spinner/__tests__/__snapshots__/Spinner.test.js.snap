// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render component 1`] = `
<div
  className="spinner-border"
  role="status"
>
  <span
    className="sr-only"
  >
    Loading...
  </span>
</div>
`;
